### PR TITLE
feat : implemented input and image components

### PR DESCRIPTION
**Pull request by deekshithmd**
implemented responsive image and input components, please review it and give your comments

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    font-size: 1.5rem;
+    padding: 0.5rem;
+}
+.error-msg{
+    color: red;
+    /* display: none; */
+}
+.success-msg{
+    color: green;
+    /* display: none; */
```
Comment (PuneetP16) [inline components/input/input.css]: Everything looks good, just remove redundant commented lines.

Comment (deekshithmd) [reply components/input/input.css]: ok, will do that thank you for the review

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.medium-label{
+    font-size: 2rem;
+}
+.primary-input{
+    padding: 1rem 1.5rem;
+    font-size: 2rem;
+    width: fit-content;
+    border-radius: 0.9rem;
```
Comment (tarunsankhla) [inline components/input/input.css]: try using px for small scale bcoz border-radius is something we usually don't prefer to enlarge or minimize

Comment (deekshithmd) [reply components/input/input.css]: ok, I will do that thank you for the review

**Pull request merged**